src/amo/routes.js

```diff
@@ -52,6 +53,7 @@ export default (
       <Route path="collections/:username/:slug/" component={Collection} />
       {config.get('enableNewCollectionsUI') && (
         <React.Fragment>
+          <Route path="collections/" component={CollectionList} />
```

Comment:
I think we need to change the dropdown menu to use this link, because currently it uses `/collections/${siteUser.username}/`.